File: src/main/java/com/example/tink_lab/models/Language.java
package com.example.tink_lab.models;

/**
 * @param code краткое название языка, используемое в качестве параметра для запроса к внешнему сервису
 * @param fullName полное название языка
 */
public record Language(String code, String fullName) {
}

File: viewConsole/src/main/kotlin/models/Language.kt
package com.example.models

import kotlinx.serialization.Serializable

/**
 * Содержит информацию о конкретном языке, используемым сервисом для перевода
 * @param code краткое название языка, используемое в качестве параметра запроса
 * @param fullName полное название языка
 */
@Serializable
data class Language(val code: String, val fullName: String)
